### PR TITLE
v4.3.0

### PR DESCRIPTION
**Pull request by TheTrunk**
- Allow WalletConnect and Metamask
  - Users can now use WalletConnect and Metamask to log into FluxOS directly using their Ethereum address as well as own, register or update applications ownership.
- Fixes v7 UI of nodes section and staticip
- Fixes ethereum regex
- Various other minor fixes

### CODE REVIEW

File: HomeUI/src/views/apps/Management.vue [hunk 1]
```diff
@@ -116,7 +116,7 @@
             />
             <list-entry
               title="Static IP"
-              :data="callResponse.datastaticip ? 'Yes, Running only on Static IP nodes' : 'No, Running on all nodes'"
+              :data="callResponse.data.staticip ? 'Yes, Running only on Static IP nodes' : 'No, Running on all nodes'"
```
Comment (Cabecinha84) [inline HomeUI/src/views/apps/Management.vue:119]: Nice catch!

**Review by Cabecinha84 — APPROVED** [review]
I don't have walletconnect or metamask, but it's only UI and the code seems good. When I am back to Lisbon I will start using those.
I think those are still missing on Xdao page (in case you want to add it for this release).
For now, ack, great job, love the logos!

**Comment by TheTrunk** [comment]
> I don't have walletconnect or metamask, but it's only UI and the code seems good. When I am back to Lisbon I will start using those. I think those are still missing on Xdao page (in case you want to add it for this release). For now, ack, great job, love the logos!

xdao is skipped as it expects zelid on the backend, can't use eth yet. Will add to board for rechecking and allowing.

**Comment by FliKites** [comment]
I can test the metamask and wallet connect once it's pushed. I have both.

**Pull request merged**